grafana/grafana/base/kustomization.yaml

```diff
@@ -10,3 +10,5 @@ resources:
 - dashboard.yaml
 - argo-dashboard.yaml
 - grafana.yaml
+- jupyterhub-sli-dashboard.yaml
```

Comment:
Could you sort the dashboards in alphabetical order? :-)

Reply:
done :)